### PR TITLE
Fix templates API versions

### PR DESCRIPTION
**Pull request by ese**
*****************************************************************************************

Thank you for contributing to kubernetes/charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/kubernetes/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/kubernetes/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the 
history. This will make it easier to identify new changes. The PR will be squashed 
anyways when it is merged. Thanks.

*****************************************************************************************

### CODE REVIEW

File: incubator/airflow/templates/statefulsets-workers.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ spec:
   ##   https://github.com/kubernetes/kubernetes/commit/c2c5051adf096ffd48bf1dcf5b11cb47e464ecdd
   podManagementPolicy: Parallel
   replicas: {{ .Values.celery.num_workers }}
+  selector:
+    matchLabels:
+      app: {{ template "airflow.name" . }}-worker
```
Comment (gsemet) [inline incubator/airflow/templates/statefulsets-workers.yaml:23]: why not airflow.fullname?

File: incubator/airflow/templates/deployments-flower.yaml [hunk 2]
```diff
@@ -15,6 +15,10 @@ spec:
     rollingUpdate:
       maxSurge: 4
       maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: {{ template "airflow.name" . }}-flower
```
Comment (gsemet) [inline incubator/airflow/templates/deployments-flower.yaml:20]: look like fullname is indeed preferable:
https://github.com/kubernetes/charts/blob/master/stable/nginx-lego/templates/nginx-service.yaml

File: incubator/airflow/templates/statefulsets-workers.yaml (hunk 1 above)
Comment (ese) [reply incubator/airflow/templates/statefulsets-workers.yaml:23]: This need to match with the pod labels and this is what is in the pod template https://github.com/gsemet/charts/pull/5/files/1ed5f6865727ae9456cfb3951386c10b69b9c5a3#diff-f93802186b180878bc908719eff78c01R31

File: incubator/airflow/templates/deployments-flower.yaml (hunk 2 above)
Comment (ese) [reply incubator/airflow/templates/deployments-flower.yaml:20]: Same as before. I don't get the relation with the `service` template in the nginx-lego chart

**Comment by gsemet** [comment]
ok thanks !

**Pull request merged**